MAUI/PDF-Viewer/Add-Remove-Modify-Annotations.md

```diff
@@ -171,4 +171,21 @@ private void OnAnnotationEdited(object sender, AnnotationEventArgs e)
     Debug.WriteLine("The {0} is edited successfully", editedAnnotation.Name);
 }
 {% endhighlight %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+### Adding custom information to a annotation
+
+The [SfPdfViewer](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html) allows you to assign custom information to annotations. The `CustomData` property is utilized to store additional information about each annotation instance for reference. However, it's essential to note that these data are solely intended for reference purposes and will not be displayed in the PdfViewer interface.
+
+The provided code sample illustrates how to set the `CustomData` property for an annotation. Within this code snippet, we set the custom data to represent the modification time of the annotation.
+
+{% tabs %}
+{% highlight C# %}
+
```

Comment:
<img width="398" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/ff412854-75ef-4dae-bb39-111d1ea43301">

As the condition handled in the added event, can we mention as **Created** date instead of **Modified** date?

Reply:
Changed.